crates/deltalake/examples/basic_operations.rs

```diff
@@ -1,3 +1,4 @@
+use std::fs;
```

Comment:
The code that follows is an informal test to write then read an encrypted parquet file. Adapted from their standard example